getit/getit.py

```diff
@@ -198,7 +200,12 @@ def main():
     system = ''
     fname = ''
     info = ''
-    path = "C:\\Users\\" + getuser() + "\\Downloads\\getit\\"
+    if (platform.system() == "Windows"):
+        path = "C:\\Users\\" + getuser() + "\\Downloads\\getit\\"
+    else:
+        path = "/Users/" + getuser() + "/Downloads/getit"
+        if not os.path.exists(path):
```

Comment:
This isn't required as i am checking the path in the next command, if it doesn't exist, it will create the path.